src/sections/SideBySide/SideBySide.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import { Box, Heading, Text, useMediaQuery } from "@chakra-ui/react";
+import { SectionButton } from "../Button/SectionButton";
+
+interface Props {
+  link: { text: string; target: string };
+  title: string;
+  content: string;
+  contentOn: "left" | "right";
+  imageUrl: string;
+}
+
+export const SideBySide = ({
+  link,
+  title,
+  content,
+  contentOn,
+  imageUrl,
+}: Props) => {
+  // TODO: Investigate if these media queries are actually working properly.
+  const [isSmallerThan1280] = useMediaQuery("(max-width: 1280px)");
+
+  const navigateTo = (target: string) => {
+    window.location.href = target;
+  };
+
+  return (
+    <>
+      {!isSmallerThan1280 && (
+        <Box
+          my="16"
+          position="relative"
+          display="flex"
+          flexDirection={contentOn === "left" ? "row-reverse" : "row"}
```

Comment:
There has to be a better, chakra recommend, way to handle this. The ternaries in the markup really bother me.

Reply:
I’ll investigate a bit